tensorboard/plugins/pr_curve/pr_curves_plugin.py

```diff
@@ -214,29 +214,42 @@ def is_active(self):
     return any(six.itervalues(all_runs))
 
   def _process_tensor_event(self, event, thresholds):
-    """Converts a TensorEvent into an dict that encapsulates information on it.
+    """Converts a TensorEvent into a dict that encapsulates information on it.
 
     Args:
       event: The TensorEvent to convert.
       thresholds: An array of floats that ranges from 0 to 1 (in that
-        direction).
+        direction). This list of thresholds is not culled. We span the whole
+        range from 0 to 1. This method filters the thresholds based on the
```

Comment:
After reading this description, I was surprised at what the code actually does. You aren't actually "removing values with TP + FP = 0"; you're only trimming such values from the tail of the data. Do you think that this merits clarification?

Reply:
That is such a clearer way of stating that.

Reply:
Also updated the PR description.